### PR TITLE
Added addLore() method

### PR DESCRIPTION
**Pull request by ghost**
## Introduction

This PR is necessary for adding lores in items rather than using a hack on setLore()

### Relevant issues

None

### API changes

` Item::addLore() `

### Behavioural changes

None

## Backwards compatibility

None

### CODE REVIEW

**Comment by Muqsit** [comment]
It would be better if the function utilizes setLore() method.

**Comment by KairusDarkSeeker** [comment]
@Musit Good idea, how do I change it?

File: src/pocketmine/item/Item.php [hunk 1]
```diff
@@ -684,6 +684,21 @@ public function getLore() : array{
...
+		if(!isset($tag->display)){
+			$tag->display = new CompoundTag("display", []);
+		}
+		
+		if(!isset($tag->display->Lore)) {
+			$tag->display->Lore = new ListTag("Lore");
+		}
+		
+		$tag->display->Lore->setTagType(NBT::TAG_String);
+		$tag->display->Lore[$tag->display->Lore->getCount()] = new StringTag("", $lore);
```
Comment (Muqsit) [inline src/pocketmine/item/Item.php]: I guess this should be
`$tag->display->Lore->{$tag->display->Lore->getCount()}`

Comment (KairusDarkSeeker) [inline src/pocketmine/item/Item.php]: Both $tag->display->Lore->{$tag->display->Lore->getCount()} and $tag->display->Lore->[$tag->display->Lore->getCount()] works

Comment (Sandertv) [inline src/pocketmine/item/Item.php]: Even though both work you're using two different methods in one line now. I'd say change it to what @Muqsit recommends.

Comment (KairusDarkSeeker) [inline src/pocketmine/item/Item.php]: @Sandertv okay

**Comment by KairusDarkSeeker** [comment]
Wow, using git on a vps is much more efficient than using the web editor

**Comment by KairusDarkSeeker** [comment]
@Muqsit Can't, I already made the PR so yeah

**Comment by thebigsmileXD** [comment]
@Muqsit https://github.com/pmmp/PocketMine-MP/commit/3e76c3a6ddd8ec9cb25a20c96bbbaadb72e4f665#diff-be0be6071c2c2f0abecaa7dd4e16e2f1R686 there it isn't utilized. is addlore actually needeed?

**Review by dktapps — CHANGES_REQUESTED** [review]
I don't see the need for all the extra (copy-pasted) code. Simply:
```
$lore = $item->getLore();
$lore[] = "new line";
$item->setLore($lore);
```

**Comment by KairusDarkSeeker** [comment]
@dktapps okay

**Pull request closed without merging**